core/src/main/java/com/arcrobotics/ftclib/hardware/SimpleServo.java

```diff
@@ -3,40 +3,79 @@
 import com.qualcomm.robotcore.hardware.HardwareMap;
 import com.qualcomm.robotcore.hardware.Servo;
 
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+
 public class SimpleServo implements ServoEx {
 
     Servo servo;
+    AngleUnit angleUnit;
+
+    //always stored internally as radians
     double maxAngle, minAngle;
+
     final double maxPosition = 1;
     final double minPosition = 0;
 
-    public SimpleServo(HardwareMap hw, String servoName) {
+    public SimpleServo(HardwareMap hw, String servoName, AngleUnit angleUnit) {
+
         servo = hw.get(Servo.class, servoName);
-        maxAngle = 180;
+        this.angleUnit = angleUnit;
+
+        maxAngle = Math.toRadians(180);
         minAngle = 0;
+
     }
 
-    public SimpleServo(HardwareMap hw, String servoName, double maxAngle, double minAngle) {
+    public SimpleServo(HardwareMap hw, String servoName){
+        this(hw, servoName, AngleUnit.DEGREES);
+    }
+
+    public SimpleServo(HardwareMap hw, String servoName, double maxAngle, double minAngle, AngleUnit angleUnit) {
+
         servo = hw.get(Servo.class, servoName);
-        this.maxAngle = maxAngle;
-        this.minAngle = minAngle;
+
+        if(angleUnit == AngleUnit.DEGREES) {
+            this.maxAngle = Math.toRadians(maxAngle);
+            this.minAngle = Math.toRadians(minAngle);
+        } else {
+            this.maxAngle = maxAngle;
+            this.minAngle = minAngle;
+        }
+
+        this.angleUnit = angleUnit;
+
     }
 
+    public SimpleServo(HardwareMap hw, String servoName, double maxAngle, double minAngle) {
+        this(hw, servoName, maxAngle, minAngle, AngleUnit.DEGREES);
+    }
 
     @Override
-    public void rotateDegrees(double angle) {
+    public void rotateAngle(double angle) {
         angle = getAngle() + angle;
         turnToAngle(angle);
     }
 
     @Override
     public void turnToAngle(double angle) {
-        if(angle > maxAngle)
-            angle = maxAngle;
-        else if(angle < minAngle)
-            angle = minAngle;
 
-        setPosition((angle - minAngle) / (getAngleRange()));
+        //use local variable in case we need to convert units to degrees
+        //(remember these values are always stored as radians internally)
+        double iMaxAngle = maxAngle;
+        double iMinAngle = minAngle;
+
+        if(angleUnit == AngleUnit.DEGREES) {
+            iMaxAngle = Math.toDegrees(iMaxAngle);
+            iMinAngle = Math.toDegrees(iMinAngle);
+        }
+
+        if(angle > iMaxAngle)
+            angle = iMaxAngle;
+        else if(angle < iMinAngle)
+            angle = iMinAngle;
```

Comment:
AngleUnit also has a static normalization method that clamps to `[-180, 180)` degrees or the comparable units in radians. You can easily simplify this to:
```suggestion
        //use local variable in case we need to convert units to degrees
        //(remember these values are always stored as radians internally)
        double iMaxAngle = angleUnit == AngleUnit.DEGREES ? AngleUnit.normalizeDegrees(angleUnit.toDegrees(maxAngle)) : maxAngle;
                                          
        double iMinAngle = angleUnit == AngleUnit.DEGREES ? AngleUnit.normalizeDegrees(angleUnit.toDegrees(minAngle)) : minAngle;
```